src/js/components/Main/Home/Home.container.jsx

```diff
@@ -87,15 +88,55 @@ const HomeContainer = ({concerts, getConcerts}) => {
   function onSearch() {
 
     if (helpList[0]) {
-      history.push('/concert/'+helpList[0].id)
+      history.push('/concert/' + helpList[0].id)
     }
   }
+
+
+  let currentListItem = 0;
+
   function keyPress(e) {
-    if(e.keyCode === 13 && helpList[0]){
-      history.push('/concert/'+helpList[0].id)
+    if (e.keyCode === 13 && helpList[0]) {
+      history.push('/concert/' + helpList[0].id)
+    }
+    if (e.keyCode === 40 && helpList[0]) {
+      liRef.current[0].setAttribute("tabindex", "0")
```

Comment:
I'm not sure changing `tabindex` dynamically is good idea, [it's not intended to change at all](https://developer.mozilla.org/ru/docs/Web/HTML/Global_attributes/tabindex). Could you please describe in a few words why do you need this?

Reply:
it is to add tabIndex for dynamic added element. without it focus gets on link in list item, but we need on list also list item.